platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## 5.0.2 - July 29, 2019
+* Fixed an issue where fill extrusion layers would be incorrectly rendered above other layers. ([#15065](https://github.com/mapbox/mapbox-gl-native/pull/15065))
+* Fixed a map update bug caused by the render tiles and the render passes becoming unsynchronized. ([#15092](https://github.com/mapbox/mapbox-gl-native/pull/15092))
+* Fixed `CustomGeometrySource` bug caused by usage of outdated tiles after a style update. ([#15112](https://github.com/mapbox/mapbox-gl-native/pull/15112))
```

Comment:
This is incorrect — there is no such (public) thing as `CustomGeometrySource` on iOS. Please check the original changelong entry for this PR and use that text.